packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -107,13 +107,24 @@ export class Utils {
       Constants.AadAppPlugin.id,
       Constants.AadAppPlugin.configKeys.teamsWebAppId
     ) as string;
-    const endpoint = Utils.getConfigValueWithValidation(
-      ctx,
-      isLocalDebug ? Constants.LocalDebugPlugin.id : Constants.FrontendPlugin.id,
-      isLocalDebug
-        ? Constants.LocalDebugPlugin.configKeys.endpoint
-        : Constants.FrontendPlugin.configKeys.endpoint
-    ) as string;
+
+    // const endpoint = Utils.getConfigValueWithValidation(
+    //   ctx,
+    //   isLocalDebug ? Constants.LocalDebugPlugin.id : Constants.FrontendPlugin.id,
+    //   isLocalDebug
+    //     ? Constants.LocalDebugPlugin.configKeys.endpoint
+    //     : Constants.FrontendPlugin.configKeys.endpoint
+    // ) as string;
+    let endpoint: string;
+    if (isLocalDebug) {
+      endpoint = Utils.getConfigValueWithValidation(
+        ctx,
+        Constants.LocalDebugPlugin.id,
+        Constants.LocalDebugPlugin.configKeys.endpoint
+      ) as string;
+    } else {
+      endpoint = getArmOutput(ctx, "frontendHosting_endpoint") as string;
```

Comment:
we need to check the feature flag first. If feature flag is disabled, we should still use original logic.

Reply:
Added the check. Thanks.